common/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 val ktorVersion = "2.0.0"
 val logbackVersion = "1.2.11"
 val logstashVersion = "7.1.1"
+val prometheusVersion = "1.8.5"
```

Comment:
Prometheus logger metrikker? Vises det i vanlige logger, eller ser man det et annet sted?

Reply:
Prometheus er ett bibliotek for metrikker, men logger ikke disse noe sted.

Biblioteket samler inn data/metrikker og eksponerer disse på ett ende (spesifisert i nais.yaml, typisk `/internal/metrics`). Når nais-plattformen ser at vi har konfigurert dette endepunktet så vil plattformen automatisk kalle endepunktet periodisk for å hente metrikkene, som så kan brukes til visualiseringer i grafana.